docs/docker_linux.rst

```diff
@@ -0,0 +1,57 @@
+Pull LISA docker image on Linux
+=====================
+
+This guide will walk you through pulling the LISA Docker image on a Linux system.
+
+Install docker on Linux
+-----------------------
+
+The first steps are to install Docker on your system.
+
+On Ubuntu, you can install Docker with the following commands:
+
+.. code:: bash
+
+   sudo apt update
+   sudo apt install docker.io -y
+
+On Azure Linux, you can install Docker with the following commands:
+
+.. code:: bash
+
+   sudo tdnf update
+   sudo tdnf install -y moby-engine moby-cli
+
+Start Docker service
+--------------------
+
+After installing Docker, you can start the Docker service with the following command:
+
+.. code:: bash
+
+   sudo systemctl start docker
+
+List docker images
+------------------
+
+You can check the available tags for the LISA Docker image by listing the tags from the Microsoft Azure Registry.
+
+.. code:: bash
+
+    curl -s https://mcr.microsoft.com/v2/lisa/runtime/tags/list
+
+Pull LISA docker image and verify installation
+-----------------------
+
+Once Docker is installed, you can pull the LISA Docker image. The latest tag will pull the most up-to-date version.
+
+.. code:: bash
+
+   docker pull mcr.microsoft.com/lisa:latest
+
+Alternatively, if the image is not available locally, running a container will automatically pull it from the registry.
+
+.. code:: bash
+
+   sudo docker run --rm -i mcr.microsoft.com/lisa/runtime:latest lisa
```

Comment:
Give an example of passing arguments.

Reply:
I will add the how to run in the run_test part.